appendix-basic65.tex

```diff
@@ -7392,7 +7392,7 @@ \subsection{RESUME}
                 {\bf RESUME} with no parameters attempts to
                 re-execute the statement that caused the error.
                 The {\bf TRAP} routine should have examined
-                and corrected the issue that cause the exception in this case.
+                and corrected the issue where the error occurred.
```

Comment:
...where the **_exception_** occurred.

Reply:
Ah ok, looks like error is the better term, so keep it here, and also update in the earlier description too :)